cmd/libsnap-confine-private/classic.c

```diff
@@ -56,3 +56,24 @@ sc_distro sc_classify_distro(void)
 		return SC_DISTRO_CLASSIC;
 	}
 }
+
+bool sc_is_debian_like(void)
+{
+	FILE *f SC_CLEANUP(sc_cleanup_file) = fopen(os_release, "r");
+	if (f == NULL) {
+           return false;
+	}
+
+	char buf[255] = { 0 };
```

Comment:
Can you please drop this part and use `sc_infofile_get_key` instead. There are some examples of how to use it in the test suite: https://github.com/snapcore/snapd/blob/30ef6dd52df387d359afdcd15f96210e6e0a1d71/cmd/libsnap-confine-private/infofile-test.c#L39